justbuild.it-ConsoleApp/src/main/java/com/isa/entity/Offer.java

```diff
@@ -0,0 +1,85 @@
+package com.isa.entity;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.isa.entity.enums.ServiceCategory;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import java.io.Serializable;
+import java.util.Date;
+import java.util.Objects;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Offer implements Serializable{
+    private long offerID;
+    private ServiceCategory serviceCategory;
+    private String offerContent;
+    private String city;
+    private User user;
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSXXX")
+    protected Date date;
```

Comment:
Lepiej byłoby użyć typu LocalDateTime, ale tę zmianę można zostawić na później, bo to się wiąże z dodatkową konfiguracją Jackson mappera 😉

Reply:
ok